### DISCUSSION TITLE
Container tag menu lists an untranslated tag

### DISCUSSION THREAD

**Discussion by HardRedux** [FakeFishGames/Barotrauma#14128] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

(I’m writing from a translator) In the boat editor, in the menu for selecting tags for a container, a tag is written instead of a name. Game without mods. At least this problem occurs in Russian. Just a visually unpleasant bug :)

### Reproduction steps

![Screenshot_4](https://github.com/FakeFishGames/Barotrauma/assets/138946919/e6c2470a-047a-481a-a734-621ef4b36f35)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens outside single player or multiplayer game modes (e.g. game launches on startup, something broken in the main menu)

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
wreckchaingunloader
```

**Comment by Regalis11** [comment]
This has been addressed since the report was created, looks like we just forgot to close the ticket. :)
